cfme/tests/containers/test_manageiq_ansible_users.py

```diff
@@ -3,13 +3,13 @@
 from cfme.configure.access_control import User
 from utils import testgen
 from utils.ansible import setup_ansible_script, run_ansible, \
-    fetch_miq_ansible_module, create_tmp_directory, remove_tmp_files
+    fetch_miq_ansible_module, install_virt_env, create_tmp_directory, remove_tmp_files
 from utils.appliance.implementations.ui import navigate_to
 from utils.version import current_version
 
 
 pytestmark = [
-    pytest.mark.uncollectif(lambda: current_version() < "5.8"),
+    pytest.mark.uncollectif(lambda: current_version() < "5.7"),
```

Comment:
Can you please elaborate what part is unneeded? We run tests on 5.6 as well on our system and I'd rather this test doesn't run on it